teams-fluid-hello-world/src/client/helloWorldTab/FluidContent.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import { Button } from "@fluentui/react-northstar";
+import { SharedMap } from "fluid-framework";
+import { diceValueKey } from "./Util";
+import { useState, useEffect } from "react";
+
+export type FluidContentProps = Readonly<{
+    fluidMap: SharedMap;
+}>
+
+export const FluidContent = (props : FluidContentProps) => {
+
+    const generateCurrentViewState = (): {color : string, content : string} => {
```

Comment:
nit verbosity since `state` in a React context means that it is for the current view:
```suggestion
    const generateState = (): {color : string, content : string} => {
```